content/about/summary/students.rst

```diff
@@ -0,0 +1,52 @@
+Students
+=========
+:slug: students
+:menuparent: about/summary
+
+.. image:: /theme/img/studex.jpg
+   :scale: 80%
+   :alt: Student Experience at OSUOSL
+
+Students are an integral part of the Oregon State University Open Source Lab.
+Students are highly involved in all our development and hosting activities,
+working in developer and system administrator positions.
+
+`Student Stories`_
+--------------------------------------
+Working on open source projects not only gives our students a great opportunity
+to use, understand and contribute to new technologies, but also puts their work
+out in the open where it can be seen and appreciated by users, developers and
+potential employers. For an idea of some of the projects OSL student employees
+have worked on, check out our `student stories`_.
+
+.. _Student Stories: /students/stories
+.. _student stories: /students/stories
+
+`Open Source on Campus`_
+------------------------------------------------
```

Comment:
OK, I can fix all these. I thought it was OK if it was at least as long as the text.